pkg/templates/templates.go

```diff
@@ -419,6 +419,9 @@ func (t *TemplateResolver) ResolveTemplate(tmplJSON []byte, context interface{})
 	var buf strings.Builder
 
 	err = tmpl.Execute(&buf, ctx)
+
+	resolvedResult.ReferencedObjects = t.referencedObjects
```

Comment:
The other assignment towards the end of this method is probably not needed then.

Reply:
Good catch! I just removed that in the latest push.